tests/test_docker_cat.py

```diff
@@ -0,0 +1,539 @@
+"""Automated integration test of Docker CAT
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the second line (D213)](https://app.codacy.com/gh/cnescatlab/docker-cat/pullRequest?prid=6253839)